Software/lib/GNSE_HAL/GNSE_hal.h

```diff
@@ -0,0 +1,53 @@
+
+/** Copyright © 2021 The Things Industries B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file GNSE_hal.h
+ *
+ * @copyright Copyright (c) 2021 The Things Industries B.V.
+ *
+ */
+
+#ifndef GNSE_HAL_H
+#define GNSE_HAL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdbool.h>
+
+/**
+ * HW aliases for the initialisation function
+ */
+typedef struct _GNSE_INIT_Cfg
+{
+    bool I2C_GNSE_INIT;
+    bool SENSORS_GNSE_INIT;
```

Comment:
The internals of the struct shouldn't be all capital unless it's an abbreviation.
```suggestion
    bool internal_sensors_init;
```
Please apply to the rest.
Please also add an one for external sensors